hw/top_englishbreakfast/util/sw_sources.patch

```diff
@@ -22,6 +22,45 @@ index 99485c5ff..1c8e29d86 100644
    // Request memory scrambling and init
    li a0, TOP_EARLGREY_SRAM_CTRL_MAIN_REGS_BASE_ADDR
    li a1, (1<<SRAM_CTRL_CTRL_RENEW_SCR_KEY_BIT)|(1<<SRAM_CTRL_CTRL_INIT_BIT)
+diff --git a/sw/device/lib/dif/dif_csrng.c b/sw/device/lib/dif/dif_csrng.c
```

Comment:
I don't think you should need to make these changes to dif_csrng{.c,.h}, right? The new DIF function doesn't seem to use anything earl grey specific.

Reply:
I think so, but `sw_sources.patch` already deletes several usages of `TOP_EARLGREY_CSRNG_BASE_ADDR`. I'm not sure if those deletions are actually required or if they are there to simplify the patch. @vogelpi would you mind taking a look?

Reply:
I also agree that there's no must for chenging the files dif_csrng{.c,.h} as soon the English breakfast doesn't use it.
So I changed the patch to let then untouched.

Reply:
I also agree that there's no must for chenging the files dif_csrng{.c,.h} as soon the English breakfast doesn't use it.
So I changed the patch to let then untouched.